uaclient/config.py

```diff
@@ -346,6 +347,20 @@ def features(self):
                 )
         return {}
 
+    @property
+    def timer(self):
```

Comment:
I think the idea here is to allow user to set the jobs time configurations through `ua config set`
We will probably need to move that logic under the `ua_config` key

Reply:
Sure, can take a look into that.
Are you thinking about something like `ua cofig set timer <job name> run_interval_seconds <value>`?

Reply:
Speaking about that, if we do not want to configure anything but the intervals, is there any major downside in going
```
timer:
    <job_name>: <interval_value>
```
for config?

Reply:
@lucasmoura @renanrodrigo I vote to avoid adding a `ua config set <timer> <runinterval>` for this release. I'd expect that custom setting timer run frequency to something slower than what our defaults are is not a highly used config operation that we expect lots of users to "want". But we can discuss this with @mitechie as to whether this use case is one that begs the CLI "sugar" for ease of set/unset.

Reply:
Maybe we could do this iteration using only the config file, and implement the commands in another PR afterwards